document/html/index.bs

```diff
@@ -0,0 +1,389 @@
+does not represent an [=ok status=], or does not match the
+`application/wasm` MIME type, the returned promise will be rejected with a [=TypeError=]; if
+compilation fails, the returned promise will be rejected with a {{CompileError}}.
+
+<div algorithm>
+The <dfn method for="WebAssembly">compileStreaming(source)</dfn> method, when invoked, returns the result of [=compile a potential WebAssembly response|compiling a potential WebAssembly response=] with |source|.
+</div>
+
+<h3 id="streaming-instantiation">`WebAssembly.instantiateStreaming`</h3>
+
+Note: This method was not included in the initial Wasm specification, so developers may need to provide a code path for the case where it is not present.
+
+<div algorithm>
+The <dfn method for="WebAssembly">instantiateStreaming(source, importObject)</dfn> method, when invoked, performs the following steps:
+
+    1. Let |promiseOfModule| be the result of [=compile a potential WebAssembly response|compiling a potential WebAssembly response=] with |source|.
+    1. Return the result of [=instantiate a promise of a module|instantiating the promise of a module=] |promiseOfModule| with imports |importObject|.
+</div>
+
+<h3 id="process-response">Compile a potential WebAssembly response</h3>
+
+Note: This algorithm accepts a {{Response}} object, or a
+    promise for one, and compiles and instantiates the resulting bytes of the response. This compilation
+    can be performed in the background and in a streaming manner. If the {{Response}} is not
+    [=CORS-same-origin=], does not represent an [=ok status=], or does not match the
+    `` `application/wasm` `` MIME type, the returned promise will be rejected with a [=TypeError=]; if
+    compilation or instantiation fails, the returned promise will be rejected with a
+    {{CompileError}} or other relevant error type, depending on the cause of failure.
+
+<div algorithm>
+To <dfn>compile a potential WebAssembly response</dfn> with a promise of a {{Response}} |source|, perform the following steps:
+
+    1. Let |returnValue| be [=a new promise=]
+    1. [=Upon fulfillment=] of |source| with value |unwrappedSource|:
+        1. Let |response| be _unwrappedSource_'s [=Response/response=].
+        1. Let |mimeType| be the result of [=extracting a MIME type=] from |response|'s [=response/header list=].
+        1. If |mimeType| is not `` `application/wasm` ``, reject |returnValue| with a [=TypeError=] and abort these substeps.
+
+        Note: extra parameters are not allowed, including the empty `` `application/wasm;` ``.
+
+        3. If |response| is not [=CORS-same-origin=], [=reject=] |returnValue| with a [=TypeError=] and abort these substeps.
+        1. If |response|'s [=response/status=] is not an [=ok status=], [=reject=] |returnValue| with a [=TypeError=] and abort these substeps.
+        1. [=Consume body|consume=] |response|'s body as an [=ArrayBuffer=], and let |bodyPromise| be the result.
```

Comment:
This isn't linking and is miscapitalized. Probably it is backwards.

Why aren't you seeing these sort of things in your Bikeshed output? They should at least be warnings if not errors.